spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -9,7 +9,7 @@ def verify_identity_with_doc_auth
     click_on 'Continue'
 
     expect(page).to have_current_path('/verify/doc_auth/agreement')
-    check 'ial2_consent_given'
+    check t('doc_auth.instructions.consent', app_name: APP_NAME)
```

Comment:
this isn't going to work, we don't have i18n available when the smoke tests run outside the rails process against the live server 😬

Reply:
> this isn't going to work, we don't have i18n available when the smoke tests run outside the rails process against the live server 😬

Good call, thanks. I've been having trouble running tests locally so I'd hoped to just run it in Circle, though I dunno that it would have failed for monitor tests anyways?

I suppose next best thing would be to just look for the English text label, or it may work to use the updated form name (`doc_auth[ial2_consent_given]`).

Reply:
Unfortunately when we run tests on a branch before it's merged, the smoke tests get run inside the same Rspec process as the rest of the tests so the i18n stuff is loaded, but when it gets run outside it does not

Reply:
Hm, not necessarily here, but maybe we could have a custom Rubocop cop that disallows `t` calls in these files. 🤔

Reply:
>this isn't going to work, we don't have i18n available when the smoke tests run outside the rails process against the live server 😬

Removed in 554691c.

> Hm, not necessarily here, but maybe we could have a custom Rubocop cop that disallows t calls in these files. 🤔

Follow-up at #5653.